storyblok/storyblok-design-system
# Bugfix - int-41 - modal fix on storybook docs

This PR fixes the modal issue on `SbUploadDialog` component, when visualizing it from storybook's docs, like this [link](https://blok.ink/?path=/docs/design-system-components-sbuploaddialog--default).

Note that you cannot interact with the elements that are behind the modal. 

How to test:

- One can clone this repository, navigate to this branch `fix/INT-41-modal-fix-on-storybook-docs`.
- Run `yarn install`
- Run `yarn storybook`
- Go to [localhost](http://localhost:6006/?path=/docs/design-system-components-sbuploaddialog--default) and see the changes
